tvshows/src/components/shared/AuthForm/EmailInput/EmailInput.tsx

```diff
@@ -0,0 +1,14 @@
+import { Input, InputGroup, InputProps } from "@chakra-ui/react";
+import { forwardRef } from "react";
+
+interface IEmailInputProps extends InputProps {}
+
+export const EmailInput = forwardRef(({ ...rest }: IEmailInputProps, ref) => {
```

Comment:
This is of questionable use because it does not add any specific email behaviour except the placeholder and type.

Reply:
I did not have any specific email behaviour in the first place